introduction_to_amazon_algorithms/xgboost_abalone/xgboost_abalone.ipynb

```diff
@@ -559,9 +566,9 @@
  "metadata": {
   "anaconda-cloud": {},
   "kernelspec": {
-   "display_name": "conda_python3",
+   "display_name": "Python 3",
    "language": "python",
-   "name": "conda_python3"
+   "name": "python3"
```

Comment:
It looks like you may have made this edit in an environment outside SageMaker notebooks.  Unfortunately, because SageMaker notebook instances don't have a "python3" kernel (instead it's called "conda_python3"), this notebook will open with an error and the user will have to select a different kernel.  We'd like to avoid that.  Can you resubmit with the kernelspec updated to its original values?  Thanks.